ivadomed/inference.py

```diff
@@ -194,7 +195,7 @@ def segment_volume(folder_model, fname_images, gpu_id=0, options=None):
 
     # Force filter_empty_mask to False if fname_roi = None
     if fname_roi is None and 'filter_empty_mask' in loader_params["slice_filter_params"]:
-        print("\nWARNING: fname_roi has not been specified, then the entire volume is processed.")
+        logger.info("\nWARNING: fname_roi has not been specified, then the entire volume is processed.")
```

Comment:
Would it be relevant to use `logger.warning` in such places?

Reply:
> Would it be relevant to use `logger.warning` in such places?

yes you are right